.github/workflows/stale.yml

```diff
@@ -0,0 +1,16 @@
+name: 'Close stale PRs'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v3
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          stale-pr-message: "This PR has been marked as stale because it's been opened for more than 30 days. If no action is taken, it'll be closed in 5 days."
```

Comment:
I agree that this is a good idea - have you considered something similar for issues? I don't think we'd want this for all issues but maybe we could scope this via tags - eg. we could close all with a tag `needs-to-reproduce` or bugs altogether because after some time they might not be relevant anymore (and the issues keep piling up, unfortunately)

Reply:
That's a good point! I'll extend the config to delete issues that need reproduction and the steps haven't been facilitated.